src/drivers/intel/baytrail/interrupt.c

```diff
@@ -102,7 +102,7 @@ uint32_t platform_interrupt_get_enabled(void)
 	return shim_read(SHIM_PIMR);
 }
 
-void platform_interrupt_mask(uint32_t irq)
+void baytrail_interrupt_mask(uint32_t irq)
```

Comment:
I thought it could be controversial. TBH, by this I made my life a bit easier. I moved byt and hsw out of the way "temporarily" to safely concentrate on the only cascading arch - cAVS. This way we're sure not to break hsw and byt on this long journey. So, would it be acceptable to do it this way, but in the end add 2 more patches to return byt and hsw back to the generic APIs? Would keep the series a bit easier really...

Reply:
We need to be consistent across all driver using the same APIs, this would just add extra unessesary APIs to learn and document.